srs_data/tagging.py

```diff
@@ -5,6 +5,7 @@
 from srs_data.constants import ASAPP_ROOT, ASAPP_MLENG_ROOT
 from srs_data.constants import CLIENT_FULL_NAMES
 from srs_data.base import BaseTool
+from srs_data.sampling import GenerateBaselineReport
```

Comment:
this doesn't seem to be in sampling. Did you mean to use `GenerateUniformSampleForClient`?

Reply:
Haha. Oops. Hmm.